docs/data-sources/fabric_node_member.md

```diff
@@ -21,17 +30,17 @@ data "aci_fabric_node_member" "example" {
 
 ## Argument Reference
 
-- `serial` - (Required) serial of Object fabric_node_member.
+- `serial` - (Required) Serial Number of the Fabric Node Member. Type: String.
 
 ## Attribute Reference
 
-- `id` - Attribute id set to the Dn of the Fabric Node Member.
-- `annotation` - (Optional) annotation for object fabric_node_member.
-- `ext_pool_id` - (Optional) ext_pool_id for object fabric_node_member.
-- `fabric_id` - (Optional) place holder for a value
-- `name_alias` - (Optional) name_alias for object fabric_node_member.
-- `node_id` - (Optional) node id
-- `node_type` - (Optional) node_type for object fabric_node_member.
-- `pod_id` - (Optional) pod id
-- `role` - (Optional) system role type
-- `serial` - (Optional) serial number
+- `id` - (Read-Only) Attribute ID set to the DN of the Fabric Node Member. Type: String.
+- `annotation` - (Read-Only) Specifies the annotation of a Fabric Node Member. Type: String.
+- `ext_pool_id` - (Read-Only) External pool ID of object Fabric Node Member. Type: String.
+- `fabric_id` - (Read-Only) Fabric ID of the Fabric Node Member. Type: String.
+- `name_alias` - (Read-Only) Name alias of object Fabric Node Member. Type: String.
+- `node_id` - (Read-Only) Node ID Number of the Fabric Node Member. Type: String.
+- `node_type` - (Read-Only) Node type of object Fabric Node Member. Type: String.
```

Comment:
why do some contain object and some don't?

Reply:
fixed